_data/projects/TellForm.yml

```diff
@@ -0,0 +1,18 @@
+name: TellForm
+desc: Free Opensource Alternative to TypeForm or Google Forms
+site: https://tellform.com
+tags:
+- oss
+- web
+- node
+- NodeJS
+- mongoose
+- MongoDB
+- forms
+- express
+- AngularJS
+- angular
+- javascript
+upforgrabs:
+  name: Up For Grabs
+  link: https://github.com/whitef0x0/tellform/issues?q=is%3Aissue+is%3Aopen+label%3A%22Up+For+Grabs%22
```

Comment:
FYI, your link on L18 will not render an issue count on the website, as we're using a certain mechanism to parse this link for what to query the Github Issues API for. 

I'd recommend using this link: `https://github.com/whitef0x0/tellform/labels/Up%20For%20Grabs`, as that's the format we're expecting.